documentation/docs/50-api-reference/10-configuration.md

```diff
@@ -253,32 +254,19 @@ See [Prerendering](/docs/page-options#prerender). An object containing zero or m
 - `crawl` — determines whether SvelteKit should find pages to prerender by following links from the seed page(s)
 - `enabled` — set to `false` to disable prerendering altogether
 - `entries` — an array of pages to prerender, or start crawling from (if `crawl: true`). The `*` string includes all non-dynamic routes (i.e. pages with no `[parameters]`, because SvelteKit doesn't know what value the parameters should have)
-- `onError`
+- `handleHttpError`
 
   - `'fail'` — (default) fails the build when a routing error is encountered when following a link
-  - `'continue'` — allows the build to continue, despite routing errors
-  - `function` — custom error handler allowing you to log, `throw` and fail the build, or take other action of your choosing based on the details of the crawl
-
-    ```js
-    import adapter from '@sveltejs/adapter-static';
-
-    /** @type {import('@sveltejs/kit').Config} */
-    const config = {
-    	kit: {
-    		adapter: adapter(),
-    		prerender: {
-    			onError: ({ status, path, referrer, referenceType }) => {
-    				if (path.startsWith('/blog')) throw new Error('Missing a blog page!');
-    				console.warn(
-    					`${status} ${path}${referrer ? ` (${referenceType} from ${referrer})` : ''}`
-    				);
-    			}
-    		}
-    	}
-    };
-
-    export default config;
-    ```
+  - `'ignore'` - silently ignore the failure and continue
+  - `'warn'` — continue, but print a warning
+  - `(details) => void` — a custom error handler that takes a `details` object with `status`, `path`, `referrer` and `referenceType` properties
```

Comment:
Should we mention that you have to throw an error from custom handler to fail the prerender process/ build?